platform/local_country_file_utils.cpp

```diff
@@ -205,11 +207,13 @@ void FindAllLocalMapsInDirectoryAndCleanup(string const & directory, int64_t ver
   }
 }
 
-void FindAllLocalMapsAndCleanup(int64_t latestVersion, vector<LocalCountryFile> & localFiles)
+void FindAllLocalMapsAndCleanup(int64_t latestVersion, vector<LocalCountryFile> & localFiles,
```

Comment:
Please fix parameters order, latestVersion must be after localFiles same as other methods.